dataflow-website/recipes/kinesisdemo/kinesisproducer/src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+
+spring:
+  cloud:
+    stream:
+      bindings:
+        output:
+          destination: test-kinesis-stream
+          group: test-kinesis-stream-group
```

Comment:
Remove this, please, from here. It's not a `producer` option. so, won't confuse end-users.

Reply:
Good catch! I removed it from the wrong app by mistake. Fixed now.